### PR TITLE
chafa 1.6.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
`chafa/chafa.h`
<details>
<summary>LICENSE header ref</summary>

```
/* -*- Mode: C; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- */

/* Copyright (C) 2018-2021 Hans Petter Jansson
 *
 * This file is part of Chafa, a program that turns images into character art.
 *
 * Chafa is free software: you can redistribute it and/or modify
 * it under the terms of the GNU Lesser General Public License as published
 * by the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 *
 * Chafa is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU Lesser General Public License
 * along with Chafa.  If not, see <http://www.gnu.org/licenses/>. */
```

</details>

File: Formula/chafa.rb [hunk 1]
```diff
@@ -1,10 +1,9 @@
 class Chafa < Formula
   desc "Versatile and fast Unicode/ASCII/ANSI graphics renderer"
   homepage "https://hpjansson.org/chafa/"
-  url "https://hpjansson.org/chafa/releases/chafa-1.4.1.tar.xz"
-  sha256 "46d34034f4c96d120e0639f87a26590427cc29e95fe5489e903a48ec96402ba3"
-  license "GPL-3.0"
-  revision 1
+  url "https://hpjansson.org/chafa/releases/chafa-1.6.0.tar.xz"
+  sha256 "0706e101a6e0e806335aeb57445e2f6beffe0be29a761f561979e81691c2c681"
+  license "GPL-3.0-or-later"
```
Comment (carlocab) [inline Formula/chafa.rb:6]:
```suggestion
  license "LGPL-3.0-or-later"
```

Comment (chenrui333) [reply Formula/chafa.rb]: good call, cannot believe i missed it 😂

**Review by carlocab — APPROVED** [review]

File: Formula/chafa.rb (hunk 1 above)
Comment (carlocab) [reply Formula/chafa.rb]: I actually didn't see it in the header ref you posted and saw that it was LGPL somewhere else. Then I re-read your header ref twice...

**Pull request closed without merging**